### PR TITLE
Support only last 3 releases

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- show only last 3 releases, rcs and master
- add tests for that functionality
- add types in `release-fetcher.ts` so that tests pass

**Related issue(s)**

#515

### CODE REVIEW

**Comment by aerfio** [comment]
image

**Comment by aerfio** [comment]
I'm basing this feature on the fact that once prereleases are filtered by `ReleaseFetcher.filterReleased`, so the only tags allowed are newer then newest release, so we just have to cut release array to 3 latest ones

File: tools/content-loader/src/github-client/github-client.ts [hunk 1]
```diff
@@ -16,19 +16,14 @@ export class GitHubClient {
...
     this.config = config;
-    this.octokit = new Octokit();
-
-    if (config.token) {
-      this.octokit.authenticate({
-        type: "token",
-        token: config.token,
-      });
-    }
+    this.octokit = new Octokit({
```
Comment (magicmatatjahu) [inline tools/content-loader/src/github-client/github-client.ts]: Please use simple if :P `if (config.token)`

File: tools/content-loader/README.md [hunk 2]
```diff
@@ -43,13 +43,14 @@ The configuration options are as follows:
 | **APP_DOCS_OUTPUT**                     | **YES**  | `docs`                   | Path for storing the documentation results                        |
 | **APP_DOCS_VERSIONS_CONFIG_FILE**       | **YES**  | `versions.json`          | Path to the website documentation configuration                   |
 | **APP_DOCS_TEMP_DIR**                   | **YES**  | `tempDocsDir`            | Path for storing temporary data for documentation                 |
+| **APP_DOCS_RELEASE_NUMBER**             | **NO**   | `3`                      | Number of latest releases to be shown                             |
```
Comment (klaudiagrz) [inline tools/content-loader/README.md]:
```suggestion
| **APP_DOCS_RELEASE_NUMBER**             | **NO**   | `3`                      | Number of the latest releases to be shown on the website                           |
```

**Pull request merged**